packages/dd-trace/src/appsec/iast/analyzers/weak-hash-analyzer.js

```diff
@@ -16,7 +16,10 @@ const EXCLUDED_LOCATIONS = [
   path.join('node_modules', 'redlock', 'dist', 'cjs'),
   path.join('node_modules', 'ws', 'lib', 'websocket-server.js'),
   path.join('node_modules', 'mysql2', 'lib', 'auth_41.js'),
-  path.join('node_modules', '@mikro-orm', 'core', 'utils', 'Utils.js')
+  path.join('node_modules', '@mikro-orm', 'core', 'utils', 'Utils.js'),
+  path.join('node_modules', 'mongodb', 'lib', 'core', 'connection', 'connection.js'),
+  path.join('node_modules', 'sqreen', 'lib', 'package-reader', 'index.js'),
+  path.join('node_modules', 'pusher', 'lib', 'utils.js')
 ]
 
```

Comment:
You may want to use [`getNodeModulesPaths`](https://github.com/DataDog/dd-trace-js/blob/32c664618225221fd75527f0855a85f3dd33a97b/packages/dd-trace/src/appsec/iast/path-line.js#L88C10-L88C29) from `path-line.js`

```suggestion
const EXCLUDED_LOCATIONS = getNodeModulesPaths(
  'etag/index.js',
  'redlock/dist/cjs',
  ...
)
```